packages/component-library/src/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,72 @@
+/** @jsx jsx */
+/* eslint-disable */
+import { jsx, css } from "@emotion/core";
+import PropTypes from "prop-types";
+import Checkbox from "@material-ui/core/Checkbox";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+
+/*
+const checkboxClass = props => css`
+  background: ${props.bkgndColor};
+  cursor: pointer;
+  border: 2px solid ${props.accentColor};
+`;
+*/
+
+const CivicCheckbox = ({
```

Comment:
```suggestion
const Checkbox = ({
```

Recommend importing the checkbox as MaterialCheckbox and having this be Checkbox